src/helix-ui/elements/_hx-accordion-panel.less

```diff
@@ -0,0 +1,40 @@
+@import "./_hx-element";
+@import "core/hx-icon";
+
+#toggle {
+  background-color: transparent;
+  border: 0;
+  cursor: pointer;
+  padding: 0;
+  text-align: left;
+  width: 100%;
+
+  &[aria-expanded="true"] {
+    .header__icon {
+      transform: scaleY(-1);
```

Comment:
Using CSS to flip the icon instead of modifying the DOM via `hx-icon[type]` change.